packages/os/early-boot-config.service

```diff
@@ -3,6 +3,9 @@ Description=Bottlerocket userdata configuration system
 # Need network online to talk to IMDS.
 After=network-online.target apiserver.service
 Requires=network-online.target apiserver.service
+# This is for ordering purposes, systemd will ignore it if the mount unit
+# never runs because the device doesn't exist
+After=media-cdrom.mount
```

Comment:
Our config services should be able to rely on this having run by this point without specifying it.

It's leaking through the `local-fs.target` abstraction, which I'd guess is because systemd doesn't know it's supposed to have started it. See my suggestion for the mount unit.